lidar_imu_correction:
  imu_topic: "/imu/data"
  lidar_topic: "/velodyne_points"
  rotation_matrix: [1, 0, 0,
                    0, 1, 0,
                    0, 0, 1]
  translation_vector: [0, 0, 0]